integration_tests/commands/websocket/setup.go

```diff
@@ -65,28 +60,28 @@ func (e *WebsocketCommandExecutor) ConnectToServer() *websocket.Conn {
 	return conn
 }
 
-func (e *WebsocketCommandExecutor) FireCommand(conn *websocket.Conn, cmd WebsocketCommand) (interface{}, error) {
-	command := []byte(cmd.Message)
+func (e *WebsocketCommandExecutor) FireCommand(conn *websocket.Conn, cmd string) interface{} {
+	command := []byte(cmd)
 
 	// send request
 	err := conn.WriteMessage(websocket.TextMessage, command)
 	if err != nil {
-		return nil, fmt.Errorf("error sending websocket request: %v", err)
```

Comment:
I agree. Since async server was written like this, I matched it. Probably in another refactor PR, we can pick up both async and websocket tests and refactor this.